sdk/core/src/main/java/com/klaviyo/coresdk/networking/requests/KlaviyoRequest.kt

```diff
@@ -14,24 +20,40 @@ internal abstract class KlaviyoRequest: NetworkRequest() {
     override var queryData: String? = null
     override var payload: String? = null
 
+    /**
+     * Adds an anonymous ID to the given property map
```

Comment:
Should we expand here to say that this is the identifier that the device will be identified by in Klaviyo if an email or phone number is not provided?

Reply:
Good idea!